internal/server/evaluation/server.go

```diff
@@ -5,7 +5,7 @@ import (
 
 	"go.flipt.io/flipt/internal/storage"
 	"go.flipt.io/flipt/rpc/flipt"
-	rpcEvalution "go.flipt.io/flipt/rpc/flipt/evaluation"
+	rpcevaluation "go.flipt.io/flipt/rpc/flipt/evaluation"
```

Comment:
```suggestion
	"go.flipt.io/flipt/rpc/flipt/evaluation"
```

can we just use `evaluation` like we do `flipt` for the `go.flipt.io/flipt/rpc/flipt` import?